packages/compiler/src/core/program.ts

```diff
@@ -576,12 +578,31 @@ export async function compile(
       },
     };
     try {
-      await emitter.emitFunction(context);
+      const result = (await emitter.emitFunction(context)) as any;
```

Comment:
do we want to have this in the compiler as the rest is still experiemental?

Reply:
Maybe we can add a compiler flag? --experimental-alloy